### PR TITLE
Adding Pinterest provider

### PR DESCRIPTION
**Pull request by eduardo-marcolino**
I had to implement this on a recent project.
Hope this helps ;)

### CODE REVIEW

**Comment by eduardo-marcolino** [comment]
As requested #646 #378

File: additional-providers/hybridauth-pinterest/Providers/Pinterest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    $data = $profile->data;
+
+    $this->user->profile->identifier = $data->id;
+    $this->user->profile->firstName = $data->first_name;
+    $this->user->profile->lastName = $data->last_name;
+    $this->user->profile->displayName = $data->username;
+    
+    if (isset($data->image->{'60x60'})) {
```
Comment (christ0ph3r) [inline additional-providers/hybridauth-pinterest/Providers/Pinterest.php]: Nice work.  I feel 60x60 is too small.

Comment (eduardo-marcolino) [inline additional-providers/hybridauth-pinterest/Providers/Pinterest.php]: Yeah me too, but that's the only size returned by the API

Comment (christ0ph3r) [inline additional-providers/hybridauth-pinterest/Providers/Pinterest.php]: Oh wow thats pretty crazy they picked such a small size.  I tried looking for it in the docs but couldn't find anything.  Did you find this in the docs or the http response?  I haven't got to try this yet but looks good to me.  Thanks for making this.  I think a lot of people will find use to this.

Comment (eduardo-marcolino) [inline additional-providers/hybridauth-pinterest/Providers/Pinterest.php]: It's not in the docs. You can see a http response sample here https://github.com/hansott/pinterest-php/blob/master/tests/Pinterest/responses/get_v1_me_url_id_username_first_name_last_name_bio_created_at_counts_image.json.

Np man, i hope it helps! =)

Comment (christ0ph3r) [inline additional-providers/hybridauth-pinterest/Providers/Pinterest.php]: Thanks for the link.  Check this out.

https://s-media-cache-ak0.pinimg.com/avatars/otthans_1409489123_60.jpg

If we strip out _60 we can get the full size.

https://s-media-cache-ak0.pinimg.com/avatars/otthans_1409489123.jpg

**Pull request merged**